src/SnapshotManager.ts

```diff
@@ -471,6 +481,17 @@ export class SnapshotManager {
       instructions.drop.push(mapping.fields[property].joinTable.name);
     }
 
+    function createForeign(mapping, property, targetMapping) {
```

Comment:
We can use that function in an other function which does more than creating foreigns, but I figured I would change the least things possibles not to break anything for the moment.